work_queue/src/bindings/python3/work_queue.binding.py

```diff
@@ -45,9 +45,13 @@ def specify_port_range(low_port, high_port):
     os.environ['TCP_HIGH_PORT'] = str(high_port)
 
 cctools_debug_config('work_queue_python')
+cctools_tmpdir = os.getenv('CCTOOLS_TEMP', 'n')
```

Comment:
Instead of 'n', let's use None. (As a value, not a string.). The below simply check `if cctools_tmpdir`.